test/unit/test-utils/pinia.js

```diff
@@ -5,6 +5,12 @@ export const createPinia = () =>
   pinia.createPinia().use(() => ({
     $nuxt: {
       $openverseApiToken: '',
+      $sentry: {
+        // eslint-disable-next-line @typescript-eslint/no-empty-function
+        captureException: () => {},
+        // eslint-disable-next-line @typescript-eslint/no-empty-function
+        captureEvent: () => {},
```

Comment:
A slightly clearer and more intentional way to deal with this "no empty function" is to add a comment inside the function body, something like this:

```suggestion
        captureException: () => {/* noop */},
        captureEvent: () => {/* noop */},
```

This presents us with the opportunity to document _why_ the function is empty rather than just silencing the linter.

However, it might make more sense to go ahead and use `jest.fn()` for both of these.
```suggestion
        captureException: jest.fn(),
        captureEvent: jest.fn(),
```

On that note: can you update the unit tests to also confirm that they call `captureEvent` as we expect, with the error object and the message?